/**/can_state_estimator_node:
  ros__parameters:
    can_interface: "can1"
    steering_conversion_factor: 15.7
    rear_axle_frame: "rear_axle"
    front_axle_frame: "front_axle"
    odom_frame: "odom"
    base_frame: "base_footprint"
